cli/githubutil.py

```diff
@@ -82,4 +86,47 @@ def release_and_prepare_next_dev_cycle(
         commit_message="Prepare next dev cycle " + next_version_dev
     )
 
+def remove_outdated_webhooks(
+    github_org_name: str,
+    github_cfg_name: str,
+    concourse_cfg_name: str,
+):
+    '''
+    Removes webhooks with query parameter 'concourse_id' whose value does not match
+    parameter 'concourse_cfg_name'. Other webhooks are not processed
```

Comment:
for users of this function, it is an irrelevant implementation detail how webhooks to rm are identified